lib/asciidoctor-revealjs/highlightjs.rb

```diff
@@ -33,7 +35,21 @@ def docinfo location, doc, opts
           else
             theme_href = "#{revealjsdir}/lib/css/zenburn.css"
           end
-          %(<link rel="stylesheet" href="#{theme_href}"#{opts[:self_closing_tag_slash]}>)
+
+          unless (asset_uri_scheme = (doc.attr 'asset-uri-scheme', 'https')).empty?
+            asset_uri_scheme = %(#{asset_uri_scheme}:)
+          end
+          # use cdn.jsdelivr.net instead of cdnjs.com because it contains the latest version of highlight.js with additional languages
```

Comment:
Version 9.18.1 is now available on cdnjs.com: https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/highlight.min.js